provider/redis_test.go

```diff
@@ -9,24 +9,29 @@ package provider
 
 import (
 	"fmt"
-	"github.com/go-redis/redis/v8"
 	"reflect"
 	"testing"
 	"time"
+
+	"github.com/redis/rueidis"
 )
 
 func Test_redisOnlineTable_Get(t *testing.T) {
 	miniRedis := mockRedis()
 	miniRedis.Addr()
-	redisClient := redis.NewClient(
-		&redis.Options{
-			Addr:     miniRedis.Addr(),
-			Password: "",
-			DB:       0,
+	redisClient, err := rueidis.NewClient(
+		rueidis.ClientOption{
+			InitAddress:  []string{miniRedis.Addr()},
+			Password:     "",
+			SelectDB:     0,
+			DisableCache: true,
```

Comment:
You should add a comment to explain this configuration in the context of the unit tests.